app/views/papers/_tablist.html.erb

```diff
@@ -28,13 +29,17 @@
     <% end %>
 
     <% if paper.links.any? %>
-      <li>
-        Links:
-        <ul>
-          <% paper.links.each do |l| %><li><%= link_to l.url, l.url, target: '_blank' %></li><% end %>
-        </ul>
-      </li>
+      <% paper.links.each do |l| %><li><%= link_to l.url, l.url, target: '_blank' %></li><% end %>
     <% end %>
+
+    <%= form_for(paper, html: {class: "form-inline"}) do |p| %>
+    <div class="form-group">
+      <%= p.fields_for :links, paper.links.build do |l| %>
+        <%= l.text_field :url, class: "form-control", size: 50%>
+      <% end %>
+      <%= p.submit 'Add Link', class: "btn btn-xs btn-primary" %>
+    <% end %>
+    </div>
```

Comment:
You don't have to do this, but tablist has pissed me off for a while. I was thinking we should just take its component pieces and embed them in the reference partial instead of adding another layer of indirection. We don't reuse the tablist anywhere, either, so...